### DISCUSSION TITLE
PHP language support - but better

### DISCUSSION THREAD

**Discussion by xtrasmal** [zed-industries/zed#14597] [Show and tell]
Hi folks, 

I've created another PHP language support package. 
This one uses Phpactor as LSP, which will give you loads of code actions, quickfixes, etc etc.

https://github.com/xtrasmal/zed-lsp-phpactor

Enjoy!!!

**Comment by mosenturm** [comment]
Can you show a zed config for using phpactor with all features?

**Reply by xtrasmal** [reply]
@mosenturm before this extension came to life, I used my own which has a readme https://github.com/xtrasmal/zed-lsp-phpactor

Check it out
